seqr/utils/search/elasticsearch/es_search.py

```diff
@@ -1533,7 +1555,7 @@ def _parse_es_sort(sort, sort_config):
 
 def _get_field_values(hit, field_configs, format_response_key=_to_camel_case, get_addl_fields=None, lookup_field_prefix='', existing_fields=None, skip_fields=None):
     return {
-        field_config.get('response_key', format_response_key(field)): _value_if_has_key(
+        field_config.get('response_key') or format_response_key(field): _value_if_has_key(
```

Comment:
It is unlikely that the `response_key` is configured to `None` or an empty string. It should be safe to keep it unchanged.

Reply:
Its not about handling Nones, its about skipping an unnecessary computation. If `response_key` is provided, theres no reason `format_response_key` needs to be a valid function for that particular field. With these updates, theres a case where `format_response_key` throws an error but theres no reason to evaluate it at all since theres a `response_key`, so this changes causes the evaluation to be skipped